web-wallet/.eslintignore

```diff
@@ -4,10 +4,15 @@ node_modules
 /.svelte-kit
 /package
 .env
-.env.*
+.env.\*
```

Comment:
Why the added escaping here?

Reply:
Not too sure. Is the `.env.` needed at all, or can we just leave the `.env` on line 6?